src/index.js

```diff
@@ -0,0 +1,238 @@
+    skyImage.alt =
+      'The sun, off in the distance, surrounded by bright blue sky';
+    skyCaption.textContent = 'Photo by Grooveland Designs';
+  } else if (selectedSky === 'cloudy') {
+    skyImage.src = 'assets/sky/daoudi-aissa-cloudy.jpg';
+    skyImage.alt = 'An overcast sky full of fluffy clouds, some ominous';
+    skyCaption.textContent = 'Photo by Daoudi Aissa';
+  } else if (selectedSky === 'stormy') {
+    skyImage.src = 'assets/sky/brandon-morgan-lightning.jpg';
+    skyImage.alt =
+      'A dark sky with one big bolt of lightning stretching toward the ground';
+    skyCaption.textContent = 'Photo by Brandon Morgan';
+  } else if (selectedSky === 'snowy') {
+    skyImage.src = 'assets/sky/jessica-fadel-snowing.jpg';
+    skyImage.alt =
+      'A snow flurry against a dark background, as though it is nighttime';
+    skyCaption.textContent = 'Photo by Jessica Fadel';
+  }
+};
+
+const updateDisplayCity = () => {
+  const displayCity = document.getElementById('display-city');
+  const userInputCity = document.getElementById('input-city').value;
+
+  // update display city text content
+  displayCity.textContent = userInputCity;
+};
+
+// function to display real time temperature
+const showCurrentTemp = () => {
+  // update state current city based on completed user input
+  state.currentCity = document.getElementById('display-city').textContent;
+
+  const currentTemp = document.getElementById('temperature');
+
+  // update display temp, sky, and landscape after retrieving weather data
+  getCurrentTemp().then(() => {
+    currentTemp.textContent = `${state.temperature}° F`;
+
+    changeTemperatureColor();
+    changeLandscape();
+    changeSky();
+  });
+};
+
+// reset display city
+const resetDisplayCity = () => {
+  const displayCity = document.getElementById('display-city');
+
+  document.getElementById('input-city').value = '';
+  displayCity.textContent = 'Chicago';
+  state.currentCity = displayCity.textContent;
+
+  showCurrentTemp();
+};
+
+const registerEventHandlers = () => {
+  const increaseTempButton = document.getElementById('increase-temp');
+  increaseTempButton.addEventListener('click', increaseTemperature);
+
+  const decreaseTempButton = document.getElementById('decrease-temp');
+  decreaseTempButton.addEventListener('click', decreaseTemperature);
+
+  const userInputCity = document.getElementById('input-city');
+  userInputCity.addEventListener('input', updateDisplayCity);
+
+  const currentTempButton = document.getElementById('current-temp-button');
+  currentTempButton.addEventListener('click', showCurrentTemp);
+
+  const resetCityButton = document.getElementById('reset-city-button');
+  resetCityButton.addEventListener('click', resetDisplayCity);
+
+  const skyDropdown = document.getElementById('sky-dropdown');
+  skyDropdown.addEventListener('change', manuallyChangeSky);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
```

Comment:
It's easy to overlook this line nestled between functions. I recommend moving it to the very bottom to keep functions together and separate from code that we immediately call and execute.